generate-release/src/main.rs

```diff
@@ -55,10 +55,18 @@ enum Commands {
         #[arg(short, long)]
         overwrite_existing: bool,
     },
+    /// Generates release notes for all PRs merged with the `C-Needs-Release-Note` label.
+    ///
+    /// This will also open an issue for each PR that doesn't have a release note.
+    /// While duplicate issues will not be opened, be sure to use the `--dry-run` flag
+    /// if you want to test the release notes generation without spamming the repo.
     ReleaseNotes {
         /// Use this if you want to overwrite existing files
         #[arg(short, long)]
         overwrite_existing: bool,
+        /// If this is set, no issues will be opened.
+        #[arg(short, long)]
+        dry_run: bool,
```

Comment:
nit: this will still overwrite any previously generated release notes, _e.g._ if you run it twice in a row. So it's not really a dry run (which normally means "no mutation").

Reply:
`--no-issues` is another option, though I don't have a strong preference.